esp/templates/program/modules/resourcemodule/timeslot_import.html

```diff
@@ -26,22 +34,24 @@ <h2>Confirm Timeslot Import</h2>
 <input type="hidden" name="program" value="{{ past_program.id }}" />
 <input type="hidden" name="start_date" value="{{ start_date }}" />
 <table align="center" cellpadding="0" cellspacing="0" width="550">
-    <tr><th colspan="2">Proposed timeslots for {{ prog.niceName }}</th></tr>
-    <tr><td colspan="2"><table cellpadding="0" cellspacing="0" width="100%">
+    <tr><th colspan="4">Proposed timeslots for {{ prog.niceName }}</th></tr>
+    <tr><td colspan="4"><table cellpadding="0" cellspacing="0" width="100%">
     <tr>
         <td class="underline"><b>Date</b></td>
         <td class="underline"><b>Start</b></td>
         <td class="underline"><b>End</b></td>
+        <td class="underline"><b>Import? <input type="checkbox" onclick=toggle(this) checked></b></td>
     </tr>
     {% for t in new_timeslots %}
     <tr>
         <td class="underline">{{ t.start.date }}</td>
         <td class="underline">{{ t.start.time }}</td>
         <td class="underline">{{ t.end.time }}</td>
+        <td class="underline"><input type="checkbox" name="to_import" value="{{ t.short_description }}_{{ t.start|safe }}" checked></td>
```

Comment:
Why is this particular field t.start marked safe while nothing else in the rest of your changes is?

Reply:
It's a date-time object, which well get auto-formatted by Django in the template, but I just want the python string representation of the object so that it matches the logic in the handler. It's definitely possible there's a more appropriate template tag, but safe seemed to work for my purposes.

Reply:
Got it.  Can you make a note of this in a comment?  It's a little bit confusing since this is not what `safe` is usually used for.

Otherwise, looks good, feel free to merge.

Reply:
Actually, now that I think about it, I'm not sure why I didn't do this in the first place, but it might just be easiest to switch to the IDs of the *old resources* that are getting copied to the new program. My original intent was to try to pick something that would be unique to the *new resources* (in this case, I assumed the combination of the short description and start time should be unique), but since we are looping over the old resources anyways, I believe I can just use the IDs, which are guaranteed to be unique. I think I originally thought of using IDs, but immediately dismissed it because they would have been the IDs of the *new resources*, which will be different when saving because the objects weren't saved the first time through.

Reply:
Ah, I remember why it wouldn't work: the IDs of the *old resources* aren't pushed to the preview page. I'll just need to add them as another attribute to the *new resources* when pushing them to the preview page (and only then).